src/main/java/lesson02/EvensRangePrinter.java

```diff
@@ -0,0 +1,28 @@
+package lesson02;
+/*
+https://docs.google.com/document/d/1WHxKj738p9XRGQbF1huwgnaavW1QgBZjkf0u_SmrkIg/edit
+Task 2 (EvensRangePrinter)
+Вывести на экран чётные числа от first до last. Если first > last, ничего не выводить.
+Ограничения: нет.
+void printEvensRange(int first, int last)
+Входные аргументы int first, int last
+Тип результата Вывод в консоль
+ */
+
+public class EvensRangePrinter {
+
+    public static void main (String[] args)  {
+        printEvensRange(ConsoleInput.inputNumberFromConsole("Please input first number: "),
```

Comment:
My personal preference for arranging long function arguments or many of them is as described here:
https://stackoverflow.com/a/6023487

I haven't found a strict rule for that in java, but you can read the thread to get your opinion on this topic.

Reply:
That was fixed